src/components/swapv2/LimitOrder/ListOrder/index.tsx

```diff
@@ -589,9 +591,9 @@ export default forwardRef<ListOrderHandle>(function ListLimitOrder(props, ref) {
                   currentOrder.filledTakingAmount,
                   currentOrder.takingAmount,
                   currentOrder.takerAssetDecimals,
-                )}% filled`
+                )}% filled.`
```

Comment:
Same as above, ``? ` Your currently existing order is ${calcPercentFilledOrder(`` wont get translated